templates/base.lds

```diff
@@ -211,23 +200,31 @@ SECTIONS
      */
 
     .itim : ALIGN(8) {
-{% if text_in_itim %}
-        /* The .text sections are placed in the ITIM to improve the performance
-         * of all post-init and non-constructor application code. Note that
-         * this may cause the program to fail to link if the ITIM is not large
-         * enough to contain the entire .text section. */
-        *(.text.unlikely .text.unlikely.*)
-        *(.text.startup .text.startup.*)
-        *(.text .text.*)
-        *(.gnu.linkonce.t.*)
-{% endif %}
         *(.itim .itim.*)
+{% block force_itim %}
```

Comment:
I assume force_itim is a just a label for the block?

Reply:
Yeah, the `force_itim` block gets substituted into the .itim section in base.lds.